BurnOutSharp/ProtectionType/Macrovision.SafeCast.cs

```diff
@@ -80,10 +82,24 @@ internal string SafeCastCheckPortableExecutable(string file, PortableExecutable
                     return "SafeCast";
             }
 
+            // Found in "32bit\Tax02\cdac14ba.dll" in IA item "TurboTax Deluxe Tax Year 2002 for Wndows (2.00R)(Intuit)(2002)(352282)".
             string name = pex.FileDescription;
             if (name?.Equals("SafeCast2", StringComparison.OrdinalIgnoreCase) == true)
                 return $"SafeCast";
 
+            // Found in hidden resource of "32bit\Tax02\cdac14ba.dll" in IA item "TurboTax Deluxe Tax Year 2002 for Wndows (2.00R)(Intuit)(2002)(352282)".
+            // TODO: Fix File Description not getting properly pulled for this executable.
+            // Known versions:
+            // 4.16.050 Windows NT 2002/04/24
+            if (name?.Equals("Macrovision RTS Service", StringComparison.OrdinalIgnoreCase) == true)
+                return $"Macrovision RTS Service " + pex.FileVersion;
```

Comment:
The `$` at the start is only needed if you're going to inline a variable, which is what you should do here anyway.

```suggestion
                return $"Macrovision RTS Service {pex.FileVersion}";
```